rqueue-core/src/main/resources/public/rqueue/js/rqueue.js

```diff
@@ -111,24 +112,167 @@ function refreshLatencyChart(chartParams, div_id) {
 //  Data Exploration
 //==================================================================
 
-function exploreData(e) {
-  var element = $(e);
+function exploreData() {
+  let element = $(this);
   dataName = element.data('name');
   dataType = element.data('type');
   dataKey = element.data('key');
 }
 
+function displayHeader(response, displayPageNumberEl, pageSize) {
+  let rows = response.rows;
+  let tableHeader = $('#table-header');
+  displayPageNumberEl.empty();
+  $('#clear-queue').hide();
+  for (let i = 0; i < response.actions.length; i++) {
+    let action = response.actions[i];
+    switch (action.type) {
+      case 'DELETE':
+        $('#clear-queue').show();
+        deleteActionMessage = action.description;
+        break;
+      default:
+        console.log("Unknown action", action)
+    }
+  }
+  if (rows.length === pageSize) {
+    $('#next-page-button').prop("disabled", false);
+  } else {
+    $('#next-page-button').prop("disabled", true);
+  }
+  if (currentPage > 0) {
+    $('#previous-page-button').prop("disabled", false);
+  } else {
+    $('#previous-page-button').prop("disabled", true);
+    tableHeader.empty();
+    let headers = response.headers;
+    for (let i = 0; i < headers.length; i++) {
+      tableHeader.append("<th>" + headers[i] + "</th>");
+    }
+  }
+}
+
+function constructTable(table, className) {
+  // table : { headers: ["item", "score"], "rows: [{"columns":[{type:, value:, meta:[]}]}] }
+  //
+  let tableEl = $('<table>').addClass(className);
+  let thead = $('<thead>');
+  let headers = table.headers;
+  let rows = table.rows;
+  for (let i = 0; i < headers.length; i++) {
+    let row = $('<th>').text(headers[i]);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/sonus21/rqueue/pullRequest?prid=7247521)
```suggestion
    let row = $("<th>").text(headers[i]);
```